### ISSUE TITLE
Question about benchmark experiments

### ISSUE THREAD

**Issue by seungeunrho** [kakaoenterprise/JORLDY#40]
Hi, I am trying to reproduce the suggested results( [benchmark](https://kepnex.notion.site/Benchmark-e20f16b4d4f84b83b490edef13226658)) using your library.

Could you please give me some detailed information for each experiment?

For example, I'd like to know 

- number of nodes 
- hyperparameters(batch size, learning rate, ...)

Thank you!

**Comment by Kyushik** [comment]
According to your suggestion, we changed our benchmark with not only results but also config, script, spec. 
Thank you for your suggestion ;)

**Comment by leonard-q** [comment]
According to your suggestion, we append config, script and spec to benchmark! 
https://kepnex.notion.site/Benchmark-e20f16b4d4f84b83b490edef13226658
Thank you so much for your suggestion! :)
